### PR TITLE
Update goentri.com.prometheus.json

### PR DESCRIPTION
**Pull request by evelyndearaujo-goentri**

### CODE REVIEW

File: goentri.com.prometheus.json [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
   "serviceName": "Prometheus",
-  "version": 1,
+  "version": 2,
   "logoUrl": "https://cdn.goentri.com/logo.svg",
   "description": "Allows user to easily set up domain using Entri",
   "syncPubKeyDomain": "goentri.com",
   "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com, app.goentri.com",
   "variableDescription": "customDomain1, customDomain2, customDomain3 and customDomain4 are the subdomain to point to websitePointsTo1 websitePointsTo2, websitePointsTo3, websitePointsTo4 respectively. hostValue and customValue is the dynamic host value assigned to the domain",
   "sharedProviderName": true,
+  "hostRequired":true,
```
Comment (pawel-kow) [inline goentri.com.prometheus.json:13]: Sure about it?
I see no CNAME on APEX or alike.

Comment (pawel-kow) [inline goentri.com.prometheus.json:13]: Sure about it?
I see no CNAME on APEX or alike.

Comment (evelyndearaujo-goentri) [reply goentri.com.prometheus.json]: Hi Pawel, thanks for checking. I just set hostRequired as false.

**Review by pawel-kow — APPROVED** [review]
lgtm

**Pull request merged**